src/aragonjs-wrapper.js

```diff
@@ -227,12 +240,35 @@ const subscribe = (
     connectedApp: null,
     connectedWorkers: workerSubscriptionPool,
 
-    apps: apps.subscribe(apps => {
+    apps: apps.subscribe(async apps => {
+      const storageApp = apps.find(({ name }) => name === 'Storage')
```

Comment:
We will likely need to change this to something safer, since the name could be easily spoofed, but for the time being is enough

Reply:
Yes, i also noticed, but it doesn't seem we have other option here right now.